Frontend/CO.CDP.OrganisationApp/Pages/Registration/JoinOrganisation.cshtml.cs

```diff
@@ -1,20 +1,25 @@
 using System.ComponentModel.DataAnnotations;
 using CO.CDP.Organisation.WebApiClient;
+using CO.CDP.OrganisationApp.Constants;
+using CO.CDP.OrganisationApp.Models;
 using Microsoft.AspNetCore.Mvc;
 using OrganisationWebApiClient = CO.CDP.Organisation.WebApiClient;
 
 namespace CO.CDP.OrganisationApp.Pages.Registration;
 
 public class JoinOrganisationModel(
     IOrganisationClient organisationClient,
-    ISession session) : LoggedInUserAwareModel(session)
+    ISession session,
+    ITempDataService tempDataService) : LoggedInUserAwareModel(session)
 {
     public OrganisationWebApiClient.Organisation? OrganisationDetails { get; set; }
 
     [BindProperty]
     [Required(ErrorMessage = "Select an option")]
     public bool Join { get; set; }
 
+    public FlashMessage NotificationBannerAlreadyMemberOfOrganisation { get { return new FlashMessage("You are already a member of this organisation. <a class='govuk-notification-banner__link' href='/organisation/" + OrganisationDetails?.Id + "'>View Organisation</a>"); } }
+
```

Comment:
One question, Can't message defined in ErrorMessageList be used here?

Reply:
Yeah. I'll put that in